### ISSUE TITLE
Save answers when using back button

### ISSUE THREAD

**Issue by slinkymanbyday** [SabatinoMasala/vue-smart-questions#1]
If a user answers Q3, goes back, the answer to Q3 is lost. Is it possible to save the state instead of clearing answers when using back?

**Comment by SabatinoMasala** [comment]
Good point, I will take a look at the state of the answers and improve it.
